examples/ase/ase_openmm_nanotube.ipynb

```diff
@@ -390,7 +370,9 @@
      "output_type": "display_data"
     }
    ],
-   "source": "from nanover.nglview import NGLClient"
+   "source": [
+    "from nglclient import NGLClient"
```

Comment:
This section above doesn't reflect the recent changes to NGLView (incorporating it into nanover) -- make sure you look at the changes that merge made to this notebook

Reply:
Have manually reverted the changes that were merged accidentally, hopefully this fixes the issue.